### PR TITLE
Update sidecar image version

### PR DESCRIPTION
**Pull request by kamolhasan**
- installer updated
- chart updated
- csi-cluster-registry removed
- csidriver added to installer

### CODE REVIEW

File: hack/deploy/controller-plugin.yaml [hunk 1]
```diff
@@ -47,19 +47,6 @@ spec:
         volumeMounts:
         - name: socket-dir
           mountPath: /var/lib/csi/sockets/pluginproxy/
-      - name: csi-cluster
```
Comment (tamalsaha) [inline hack/deploy/controller-plugin.yaml]: Do to you need to add the CSIDriver object manually?

Comment (kamolhasan) [reply hack/deploy/controller-plugin.yaml]: Yes, I need to deploy `CSIDriver` and `CSINode` objects manually

Comment (tamalsaha) [reply hack/deploy/controller-plugin.yaml]: So, add them in the installer or inside the CSI driver?

File: chart/csi-vault/templates/csi-driver.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+apiVersion: storage.k8s.io/v1beta1
+kind: CSIDriver
+metadata:
+    name: {{ .Values.driverName }}
+spec:
+    attachRequired: {{ .Values.csidriver.attachRequired }}
+    podInfoOnMount: {{ .Values.csidriver.podInfoOnMount }}
```
Comment (tamalsaha) [inline chart/csi-vault/templates/csi-driver.yaml]: I don't know if `.Values.csidriver.podInfoOnMount` should be a variable. CSI driver will not work without this, right?

**Comment by tamalsaha** [comment]
- [ ] How many statefulsets should we have? 
https://kubernetes.slack.com/archives/C09QZFCE5/p1563916744026500

- [ ] We are missing liveness probe in the DaemonSet
https://github.com/kubernetes-csi/csi-driver-host-path/blob/master/deploy/kubernetes-1.15/hostpath/csi-hostpath-plugin.yaml#L106

File: chart/csi-vault/templates/csi-driver.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+apiVersion: storage.k8s.io/v1beta1
+kind: CSIDriver
+metadata:
+    name: {{ .Values.driverName }}
+spec:
+    attachRequired: {{ .Values.csidriver.attachRequired }}
```
Comment (tamalsaha) [inline chart/csi-vault/templates/csi-driver.yaml]: `attachRequired: false`

https://kubernetes-csi.github.io/docs/skip-attach.html

File: hack/deploy/controller-plugin.yaml (hunk 1 above)
Comment (kamolhasan) [reply hack/deploy/controller-plugin.yaml]: CSI driver added to installer

File: chart/csi-vault/templates/csi-driver.yaml (hunk 2 above)
Comment (kamolhasan) [reply chart/csi-vault/templates/csi-driver.yaml]: Yes, `podInfoOnMount` must be true. This value is set to true.

File: chart/csi-vault/templates/csi-driver.yaml (hunk 3 above)
Comment (kamolhasan) [reply chart/csi-vault/templates/csi-driver.yaml]: fixed

**Pull request merged**